packages/jupyter-widgets/src/manager/widget-manager.ts

```diff
@@ -86,6 +92,8 @@ export class WidgetManager extends base.ManagerBase<DOMWidgetView> {
           `Class ${className} not found in module ${moduleName}@${moduleVersion}`
         );
       }
+    }).catch(function(err: Error) {
+        console.warn(err.message);
```

Comment:
I thought having this additional check would help us avoid any uncaught promise rejections which might cause the app to crash. So, this was an added layer of good to checks I think. Earlier, I believe there was no explicit catch because all the modules were resolved locally